src/main/java/calculator/Operation.java

```diff
@@ -0,0 +1,24 @@
+package calculator;
+
+import java.util.function.BiFunction;
+
+enum Operation {
+
+    PLUS("+", (a, b) -> (a + b)),
+    MINUS("-", (a, b) -> (a - b)),
+    MULTIPLY("*", (a, b) -> (a * b)),
+    DIVIDE("/", (a, b) -> (a / b));
+
+    public final String operator;
+    private final BiFunction<Integer, Integer, Integer> function;
```

Comment:
```suggestion
    private final BinaryOperator<Integer> function;
```
인자와 결과값의 타입이 같으니까 BinaryOperator 으로 처리해 볼 수도 있겠네요 😄

Reply:
BinaryOperator가 있었군요! 감사합니다